### PR TITLE
Fix form & add fundings

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**

### CODE REVIEW

File: libs/organization/campaign/src/lib/form/campaign.shell.config.ts [hunk 1]
```diff
@@ -25,7 +26,13 @@ export class CampaignShellConfig implements FormShellConfig<CampaignControls, Ca
 
   async onSave(options: FormSaveOptions): Promise<any> {
     const id: string = this.route.getParams('movieId');
-    await this.service.save(id, this.form.value);
+
+    // TODO (DON'T LET ME MERGE): extract media from form value
```
Comment (fritzschoff) [inline libs/organization/campaign/src/lib/form/campaign.shell.config.ts:30]: okay, I won't

**Review by fritzschoff — CHANGES_REQUESTED** [review]

File: libs/movie/src/lib/movie/form/main/main.component.html [hunk 2]
```diff
@@ -174,7 +174,7 @@ <h3>General Information</h3>
     <mat-divider></mat-divider>
 
     <section id="directors">
-      <h3>Directors</h3>
+      <h3>Director(s)*</h3>
```
Comment (fritzschoff) [inline libs/movie/src/lib/movie/form/main/main.component.html:177]: Either just use the plural or use the plural pipe and check for the array length. The parenthesis is really not a good solution

File: libs/organization/campaign/src/lib/form/fundings/fundings.component.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+section {
+  margin-bottom: 32px;
```
Comment (fritzschoff) [inline libs/organization/campaign/src/lib/form/fundings/fundings.component.scss:2]: why do you need that here? In the form we are not setting this all the time. It makes the design in consistent

**Review by fritzschoff — APPROVED** [review]

File: libs/movie/src/lib/movie/form/main/main.component.html (hunk 2 above)
Comment (GrandSchtroumpf) [reply libs/movie/src/lib/movie/form/main/main.component.html:177]: This is the design from Anastasiia

Comment (fritzschoff) [reply libs/movie/src/lib/movie/form/main/main.component.html:177]: and also from Mathilde but we said we are using either the plural pipe here or using the plural right away!

File: libs/organization/campaign/src/lib/form/fundings/fundings.component.scss (hunk 3 above)
Comment (GrandSchtroumpf) [reply libs/organization/campaign/src/lib/form/fundings/fundings.component.scss:2]: The design is already not entirely consistent. The section were glued to each other. We'll have a full checkup around December with better design guidelines once everything is done

**Pull request merged**